src/Svg/Surface/SurfaceCpdf.php

```diff
@@ -217,7 +217,11 @@ function image($img, $x, $y, $w, $h, $resolution = "normal")
             case IMAGETYPE_GIF:
             case IMAGETYPE_BMP:
                 // @todo use cache for BMP and GIF
-                $img = $this->_convert_gif_bmp_to_png($img, $type);
+                if (method_exists($this, '_convert_to_png')) {
+                    $img = $this->_convert_to_png($img, $type);
+                } elseif (method_exists($this, '_convert_gif_bmp_to_png')) {// Old CPDF.php adapters
+                    $img = $this->_convert_gif_bmp_to_png($img, $type);
+                }
```

Comment:
All this would do is prevent us from throwing an exception. `$this` will never contain the cited functions unless they are actually written. I am making changes to address this deficiency outside this PR.